ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -1394,175 +1354,19 @@ extension BrowserViewController: WKUIDelegate {
     alertController.dismissedWithoutAction = {
       decisionHandler(.prompt)
     }
-    if webView.fullscreenState == .inFullscreen || webView.fullscreenState == .enteringFullscreen {
-      webView.closeAllMediaPresentations {
-        self.present(alertController, animated: true)
-      }
-      return
-    }
```

Comment:
This is done by chromium already https://source.chromium.org/chromium/chromium/src/+/main:ios/web/web_state/ui/crw_wk_ui_handler.mm;l=124